CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * None
 
 ### Fixed
+* Values of properties are replaced by default value when updating. ([#6129](https://github.com/realm/realm-js/issues/6129), since v12.0.0)
```

Comment:
```suggestion
* Fixed values of properties being replaced by default value when updating. ([#6129](https://github.com/realm/realm-js/issues/6129), since v12.0.0)
```